external-import/hunt-io/src/external_import_connector/config_variables.py

```diff
@@ -0,0 +1,56 @@
+import os
+from pathlib import Path
+
+import yaml
+from pycti import get_config_variable
+
+
+class ConfigConnector:
+    def __init__(self):
+        """
+        Initialize the connector with necessary configurations
+        """
+
+        # Load configuration file
+        self.load = self._load_config()
+        self._initialize_configurations()
+
+    @staticmethod
+    def _load_config() -> dict:
+        """
+        Load the configuration from the YAML file
+        :return: Configuration dictionary
+        """
+        config_file_path = Path(__file__).parents[1].joinpath("config.yml")
+        config = (
+            yaml.load(open(config_file_path), Loader=yaml.FullLoader)
+            if os.path.isfile(config_file_path)
+            else {}
+        )
+
+        return config
+
+    def _initialize_configurations(self) -> None:
+        """
+        Connector configuration variables
+        :return: None
+        """
+        # OpenCTI configurations
+        self.duration_period = get_config_variable(
+            "CONNECTOR_DURATION_PERIOD",
+            ["connector", "duration_period"],
+            self.load,
+        )
+
+        # Connector extra parameters
+        self.api_base_url = get_config_variable(
+            "CONNECTOR_HUNT_UI_API_BASE_URL",
+            ["connector_hunt-io", "api_base_url"],
```

Comment:
@m4r35 Could you correct the configuration and replace"-" by "_"? Thank you :)

```suggestion
            ["connector_hunt_io", "api_base_url"],
```